es6/README.md

```diff
@@ -0,0 +1,403 @@
+# Linting
+
+Included is a default ESLint configuration, `mobify-es6.yml` written for ESLint 3.x. It requires the additional modules:
+
+- `eslint-plugin-import`
+- `eslint-import-resolver-webpack`
```

Comment:
Can we have these be dependencies of `mobify-code-style` so we don't have to add them to the projects that want es6 linting?